client/coral-embed-stream/src/tabs/stream/containers/CommentBox.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { gql } from 'react-apollo';
+import { getSlotFragmentSpreads } from 'coral-framework/utils';
+import withFragments from 'coral-framework/hocs/withFragments';
+import { getDefinitionName } from 'coral-framework/utils';
 import t, { timeago } from 'coral-framework/services/i18n';
 import { can } from 'coral-framework/services/perms';
 import { isSuspended } from 'coral-framework/utils/user';
-
+import DraftArea from './DraftArea';
 import Slot from 'coral-framework/components/Slot';
 import { connect } from 'react-redux';
 import { CommentForm } from '../components/CommentForm';
 import { notifyForNewCommentStatus } from '../helpers';
+import { compose } from 'react-apollo';
 
 // TODO: (kiwi) Need to adapt CSS classes post refactor to match the rest.
-export const name = 'talk-plugin-commentbox';
+export const name = 'talk-commentbox';
```

Comment:
I didn't rename it yet, because I scared to break some existing styling adaptions of newsrooms. Kim advised against changing the `css` names and do them all at once, e.g. when we make a major release.

Reply:
Yep that's a good idea

Reply:
@kgardnr Cause it was your idea ;-)